page.go

```diff
@@ -218,11 +217,12 @@ func readPages(pages Pages, read chan<- asyncPage, seek <-chan int64, done <-cha
 			}:
 			case rowIndex := <-seek:
 				version++
-				err = pages.SeekToRow(rowIndex)
-			}
-			if err == nil {
-				break
+				Release(page)
+				if err = pages.SeekToRow(rowIndex); err != nil {
```

Comment:
How does `page` get repopulated here? Do we need a label on the outer loop that we can reference in continue?

Reply:
🤔 good call, see [35d9563](https://github.com/parquet-go/parquet-go/pull/221/commits/35d95634005697d2e1453a756d395159b1ba2c24)